client/client.go

```diff
@@ -340,6 +373,40 @@ func (c *Client) loadTargets(targets data.TargetFiles) {
 	}
 }
 
+func (c *Client) updateKeys(rootJSON []byte) error {
+	// unmarshal root.json without verifying as we need the root
+	// keys first
+	s := &data.Signed{}
+	if err := json.Unmarshal(rootJSON, s); err != nil {
+		return err
+	}
+	root := &data.Root{}
+	if err := json.Unmarshal(s.Signed, root); err != nil {
+		return err
+	}
+	c.db = verify.NewDB()
+	for id, k := range root.Keys {
+		if err := c.db.AddKey(id, k); err != nil {
+			// FIXME(TUF-0.9) Ignore unknown keyids, which can happen
```

Comment:
Learning question, because I am new to TUF. In the other PR (https://github.com/theupdateframework/go-tuf/pull/123) we remove support for TUF-0.9, but on the same time we are introducing a change specific to TUF-0.9 in this line. I am a little bit worried about consistence here. Can you explain your thoughts about this? :)

Reply:
Thanks for the review! This code was actually moved from the `getLocalMeta` function, and so the comment came along with it. However, now that https://github.com/theupdateframework/taps/blob/master/tap12.md is under consideration, I think the safest thing is that we should be ignoring unknown ids in general, in order to give us the flexibility to change the keyid algorithm in the future. We don't have any test coverage in this area, so I'll factor this bit out into a small patch in order to make this easier to review.

Reply:
Sounds good!